### PR TITLE
DFJK controls.txt

### PR DESCRIPTION
**Pull request by mariadachef**

### CODE REVIEW

**Comment by VMGuy23** [comment]
i use a and s
also there will be custom controls in next update
and dfjk might not be botable

**Comment by KaidenOnYT** [comment]
Actually dfjk is super easy you should try it!

**Comment by Hendicted** [comment]
1. that doesnt change anything
2. everyone is gonna be forced to use dfjk
no

**Comment by VMGuy23** [comment]
> Actually dfjk is super easy you should try it!

I use my own layout. I can use d and f instead of a and s, but I'm not sure what @KadeDev's FNFBot uses so I might not be able to bot the game.

**Comment by Michael82548** [comment]
Thanks. I couldn't get the options menu to show up or even be able to read the controls.txt file

this fixes everything for me (probably not for some), but it works better right now for me.

**Comment by arceuss** [comment]
ng version already has customizing controls idk what this is useful for

**Comment by VMGuy23** [comment]
> ng version already has customizing controls idk what this is useful for

This is a valid point.

**Comment by ACardboardBox1** [comment]
> 
> 
> ng version already has customizing controls idk what this is useful for

plus im pretty sure this does nothing, since the controls are hard coded in the files anyway

**Comment by VMGuy23** [comment]
> > ng version already has customizing controls idk what this is useful for
> 
> plus im pretty sure this does nothing, since the controls are hard coded in the files anyway

This _is_ the file it's coded into

**Comment by ACardboardBox1** [comment]
> 
> 
> > > ng version already has customizing controls idk what this is useful for
> > 
> > 
> > plus im pretty sure this does nothing, since the controls are hard coded in the files anyway
> 
> This _is_ the file it's coded into

_oh_

**Comment by prokube** [comment]
controls.txt is never mentioned lol this does nothing

**Comment by DamianBossPL** [comment]
just use kade engine....

**Comment by y579** [comment]
> > Actually dfjk is super easy you should try it!
> 
> I use my own layout. I can use d and f instead of a and s, but I'm not sure what @KadeDev's FNFBot uses so I might not be able to bot the game.

It uses the arrows

File: assets/preload/data/controls.txt [hunk 1]
```diff
@@ -1,17 +1,17 @@
 setUP
-W
+F
```
Comment (PoketrainerMS) [inline assets/preload/data/controls.txt:2]: you mixed up up and down if you are doing `df jk`, then it should have `j` for `up`, and `f` for `down`

File: assets/preload/data/controls.txt [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 setUP (ALTERNATE)
 FlxKey.UP
 setDOWN
-S
+J
```
Comment (PoketrainerMS) [inline assets/preload/data/controls.txt:6]: same as above goes for this.
it should be `f` here instead

**Comment by s-zenmode** [comment]
everyone will be forced to use dfjk, plus the NG release already has controls stuff

File: assets/preload/data/controls.txt [hunk 3]
```diff
@@ -29,4 +29,4 @@ ENTER
 setRESET
 R
 setRESET (ALTERNATE)
-null
\ No newline at end of file
+null
```
Comment (polypoyo) [inline assets/preload/data/controls.txt:32]: > ```diff
> - null
> + null
> ```
HOW

**Pull request closed without merging**